Cargo.toml

```diff
@@ -13,7 +13,7 @@ documentation = "http://danburkert.github.io/lmdb-rs/lmdb/index.html"
 
 [dependencies.lmdb-sys]
 path = "lmdb-sys"
-version = "0.5.0"
+version = "0.5.1"
```

Comment:
leave this out; I'll bump it appropriately during the next release (it's likely going to be a minor instead of patch release).